Brenna Chen - Sprint #3 Pull Request

brennachen opened the pull request:
Implemented 'Creating video' screen
- created 'VideoLoadingScreen.tsx' on CreateScreen
- Used React Native Animated Ellipsis to create animated triple dots
- changed style using 'styles.tsx' on CreateScreen
![IMG_9356](https://user-images.githubusercontent.com/29928553/70958045-dc022780-202c-11ea-8499-fabffbb6d0a9.PNG)

apetranik commented on src/screens/Home/HomeScreen/HomeScreen.tsx:
```diff
@@ -48,12 +43,12 @@ export default function HomeScreen() {
+              title={item.text}
               selected={selected === item.id}
               onSelect={() => setSelected(item.id)}
             />
```
Read my overview comment for this PR, I reference this file

brennachen replied:
Hi aliya,

Sorry, I was unclear on what this meant!

Thanks,

Brenna

apetranik commented on src/screens/Home/CreateScreen/VideoLoadingScreen.tsx:
```diff
@@ -0,0 +1,24 @@
+} from 'react-native'
+import styles from './styles'
+
+export default function CreateScreen() {
```
Can you rename this screen `RenderingVideoScreen` in 4 places
1. Rename the folder it's in from CreateScreen to `RenderingVideoScreen`
2. Rename filename 
3. Rename function name 
4. Update CreateScreen references in MainTabNavigator to RenderingVideoScreen

apetranik commented on src/screens/Home/CreateScreen/VideoLoadingScreen.tsx:
```diff
@@ -0,0 +1,24 @@
+    </View>
+  )
+}
+
```
The spacing on this file and a few others looks a bit off.
We have a standardized linter that enforces a common styling across all our code. Makes it easier to read for everyone and makes our code more professional.

A simple way to run ours is in terminal, `yarn lint --fix`. It will fix all of the spacing issues. You may be left with some linter warnings about unused variables - you can ignore them.

brennachen commented:
Hi Aliya,

I'm really sorry for the inconvenience I caused by failing to merge my
changes earlier.

I also ran into an issues while trying to rebase:

Using index info to reconstruct a base tree...

A .env

Falling back to patching base and 3-way merge...

CONFLICT (modify/delete): .env deleted in HEAD and modified in update.
Version update of .env left in tree.

error: Failed to merge in the changes.


I'm not sure how to proceed, and it still failed after I deleted my .env
file.


Thanks,


Brenna

On Mon, Dec 16, 2019 at 8:37 PM Aliya Petranik <notifications@github.com>
wrote:

> *@apetranik* requested changes on this pull request.
>
> Ellipsis looks good - nice and simple :)
>
> BIG NOTE:
> You edited HomeScreen in your last pull request and didn't merge that into
> master. Therefore your changes this time weren't up to date and will
> require a rebase. Plus, others are modifying HomeScreen this ticket meaning
> they weren't able to pull master and get your changes from last PR. This
> means they'll also have to do a rebase. Yours is quite complex (it's
> actually a 3-way merge conflict) so make the changes I said and then let me
> fix them and merge it for you.
>
> In the future
> Get your old tickets merged into master before starting new ones
> Pull master before you start a new ticket
> Pull master often and before you create a PR
> ------------------------------
>
> In src/screens/Home/HomeScreen/HomeScreen.tsx
> <https://github.com/ctcusc/gladeo/pull/10#discussion_r358590537>:
>
> >            renderItem={({ item }) => (
>              <Item
>                id={item.id}
> -              title={item.title}
> +              title={item.text}
>                selected={selected === item.id}
>                onSelect={() => setSelected(item.id)}
>              />
>
> Read my overview comment for this PR, I reference this file
> ------------------------------
>
> In src/screens/Home/CreateScreen/VideoLoadingScreen.tsx
> <https://github.com/ctcusc/gladeo/pull/10#discussion_r358591212>:
>
> > @@ -0,0 +1,24 @@
> +import React from 'react'
> +import AnimatedEllipsis from 'react-native-animated-ellipsis';
> +import {
> +  Text,
> +  View,
> +} from 'react-native'
> +import styles from './styles'
> +
> +export default function CreateScreen() {
>
> Can you rename this screen RenderingVideoScreen in 4 places
>
>    1. Rename the folder it's in from CreateScreen to RenderingVideoScreen
>    2. Rename filename
>    3. Rename function name
>    4. Update CreateScreen references in MainTabNavigator to
>    RenderingVideoScreen
>
> ------------------------------
>
> In src/screens/Home/CreateScreen/VideoLoadingScreen.tsx
> <https://github.com/ctcusc/gladeo/pull/10#discussion_r358592917>:
>
> > +
> +export default function CreateScreen() {
> +  return (
> +    <View style={styles.container}>
> +        <View style={styles.main}>
> +            <AnimatedEllipsis numberOfDots={4}
> +                  minOpacity={0.5}
> +                  animationDelay={200}
> +                  style={styles.ellipses}
> +            />
> +            <Text style={styles.text}>CREATING YOUR VIDEO</Text>
> +        </View>
> +    </View>
> +  )
> +}
> +
>
> The spacing on this file and a few others looks a bit off.
> We have a standardized linter that enforces a common styling across all
> our code. Makes it easier to read for everyone and makes our code more
> professional.
>
> A simple way to run ours is in terminal, yarn lint --fix. It will fix all
> of the spacing issues. You may be left with some linter warnings about
> unused variables - you can ignore them.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ctcusc/gladeo/pull/10?email_source=notifications&email_token=AHEKY2MP32LWFBZY63I2EHTQZBJPZA5CNFSM4J3UP6E2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCPMXQXI#pullrequestreview-333019229>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHEKY2OT4CYLRFNKZYO5NALQZBJPZANCNFSM4J3UP6EQ>
> .
>

apetranik commented on src/navigation/MainTabNavigator.tsx:
```diff
@@ -6,7 +6,7 @@ import { createStackNavigator, createBottomTabNavigator } from 'react-navigation
 import HomeScreen from '../screens/Home/HomeScreen/HomeScreen'
 import OnboardingScreen from '../screens/Home/OnboardingScreen/OnboardingScreen'
-import CreateScreen from '../screens/Home/CreateScreen/CreateScreen'
+import CreateScreen from '../screens/Home/RenderingVideoScreen/RenderingVideoScreen'
```
can you import it as RenderingVideoScreen here and then update it on line 36. This screen will now live in the "Create" stack, which is just all the screens that are in the 2nd tab of our app, so it will grow over time. For the merge conflict, I will fix it after you push these changes!
